DatabaseService/docker_context/apiListeners.py

```diff
@@ -1,19 +1,21 @@
 from sanic.log import logger
 from handlers.PostgresHandler import PostgresHandler
+from handlers.MongoHandler import MongoHandler
 from ServiceLogic import ServiceLogic
 
-
 async def initDbConnections(app, loop):
     logger.debug('Creating database connections')
     app.config['psqlHandler'] = PostgresHandler(loop)
+    app.config['mongoHandler'] = MongoHandler(loop)
     await app.config['psqlHandler'].initializePool()
-
+    
 def initServiceLogic(app, loop):
     logger.debug('Creating api logic handler')
     app.config['logic'] = ServiceLogic(
-        app.config['psqlHandler']
+        app.config['psqlHandler'],
+        app.config['mongoHandler']
     )
 
 async def closeDbConnections(app, loop):
     logger.debug('Closing database connections')
-    await app.config['psqlHandler'].closeConnection()
\ No newline at end of file
+    await app.config['psqlHandler'].closeConnection()

```

Comment:
we don't need to close any mongo connections?

Reply:
Added mongo connections cleanup. Resolving comment.